Doc/library/pyexpat.rst

```diff
@@ -614,6 +614,15 @@ otherwise stated.
 
 .. method:: xmlparser.ExternalEntityRefHandler(context, base, systemId, publicId)
 
+   .. warning::
+
+      Implementing a handler that accesses local files and/or the network
+      may create a vulnerability to
+      `external entity attacks <https://en.wikipedia.org/wiki/XML_external_entity_attack>`_
+      if :class:`xmlparser` is used with user-provided XML content.
```

Comment:
There is no Sphinx link to xmlparser:

```suggestion
      if :class:`!xmlparser` is used with user-provided XML content.
```

Reply:
`xmlparser` is referred many times in this file. The right solution is to add an anchor for it. It was difficult to do earlier, but it is possible in the current Sphinx.

Reply:
I'm reading @serhiy-storchaka 's reply as "does not need a fix". Please let me know if I'm reading it wrong.

Reply:
Actually, things are more interesting. I'll open a separate issue. Don't change it here.

Reply:
See #140771.